setup.py

```diff
@@ -5,8 +5,8 @@
 
 from setupbase import (
     create_cmdclass, install_npm, ensure_targets,
-    find_packages, combine_commands, ensure_python,
-    get_version, HERE
+    combine_commands, ensure_python, get_version,
+    HERE, which
```

Comment:
Could you remove `which` as it's unused?

Reply:
Absolutely

Reply:
Done!